Gravity/Gravity.Test.Unit/App.config

```diff
@@ -1,9 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
   <appSettings>
-    <add key="SQLServerAddress" value=""/>
-    <add key="SQLUsername" value=""/>
-    <add key="SQLPassword" value=""/>
-    <add key="GravityWorkspaceID" value=""/>
+    <add key="SQLServerAddress" value="tsdrelatest02.cloudapp.net"/>
```

Comment:
[Must-fix] To fix this, you should overwrite your old commit with interactive rebase and force push, not just push a new one

Reply:
I will have a look at how to do it in a moment..